services/src/main/java/org/keycloak/services/resources/WelcomeResource.java

```diff
@@ -145,7 +148,6 @@ public Response createUser() {
             expireCsrfCookie();
 
             shouldBootstrap.set(false);
-            ServicesLogger.LOGGER.createdTemporaryAdminUser(username);
```

Comment:
Removed as it's already logged in the `applianceBootstrap.createMasterRealmUser` call chain.